fineract-provider/src/main/java/org/apache/fineract/organisation/tasks/api/TasksApiResource.java

```diff
@@ -0,0 +1,171 @@
+    public String retrieveAll(@Context final UriInfo uriInfo, @QueryParam("status") @Parameter(description = "status") final Long status) {
+
+        this.context.authenticatedUser().validateHasReadPermission(this.resourceNameForPermissions);
+
+        final Collection<TaskData> staff;
+
+        staff = this.readPlatformService.retrieveAllTasks(status);
+
+        final ApiRequestJsonSerializationSettings settings = this.apiRequestParameterHelper.process(uriInfo.getQueryParameters());
+        return this.toApiJsonSerializer.serialize(settings, staff, this.responseDataParameters);
+    }
+
+    @POST
+    @Consumes({ MediaType.APPLICATION_JSON })
+    @Produces({ MediaType.APPLICATION_JSON })
+    @Operation(summary = "Create a staff member", description = "Creates a staff member.\n" + "\n" + "Mandatory Fields: \n"
+            + "officeId, firstname, lastname\n" + "\n" + "Optional Fields: \n" + "isLoanOfficer, isActive")
+    @RequestBody(required = true, content = @Content(schema = @Schema(implementation = TasksApiResourceSwagger.PostStaffRequest.class)))
+    @ApiResponses({
+            @ApiResponse(responseCode = "200", description = "OK", content = @Content(schema = @Schema(implementation = TasksApiResourceSwagger.CreateStaffResponse.class))) })
+    public String create(@Parameter(hidden = true) final String apiRequestBodyAsJson) {
+
+        final CommandWrapper commandRequest = new CommandWrapperBuilder().createTask().withJson(apiRequestBodyAsJson).build();
+
+        final CommandProcessingResult result = this.commandsSourceWritePlatformService.logCommandSource(commandRequest);
+
+        return this.toApiJsonSerializer.serialize(result);
+    }
+
+    @GET
+    @Path("{staffId}")
+    @Consumes({ MediaType.APPLICATION_JSON })
+    @Produces({ MediaType.APPLICATION_JSON })
+    @Operation(summary = "Retrieve a Staff Member", description = "Returns the details of a Staff Member.\n" + "\n" + "Example Requests:\n"
+            + "\n" + "staff/1")
+    @ApiResponses({
+            @ApiResponse(responseCode = "200", description = "OK", content = @Content(schema = @Schema(implementation = TasksApiResourceSwagger.RetrieveOneResponse.class))) })
+    public String retrieveOne(@PathParam("staffId") @Parameter(description = "staffId") final Long staffId,
+            @Context final UriInfo uriInfo) {
+
+        this.context.authenticatedUser().validateHasReadPermission(this.resourceNameForPermissions);
+
+        final ApiRequestJsonSerializationSettings settings = this.apiRequestParameterHelper.process(uriInfo.getQueryParameters());
+
+        TaskData staff = this.readPlatformService.retrieveTask(staffId);
+        if (settings.isTemplate()) {
+            // Collection<CodeValueData> taskStatus =
+            // this.codeValueReadPlatformService.retrieveCodeValuesByCode("taskStatus");
```

Comment:
This is a new class but you have commented-out code. Do you plan to make use of this later?